mevislab.github.io/content/tutorials/image_processing/image_processing3.md

```diff
@@ -14,4 +14,62 @@ menu:
 
 # Example 3: Region Growing
 ## Introduction
+A very simple approach to segment parts of an image is the region growing method. A general explanation can be found [here]("https://en.wikipedia.org/wiki/Region_growing").
 
+In this example, we want to segment the brain of an image and show the segmentation results as overlay on the original image.
+
+## Steps to do
+### Develop your network
+Add a `LocalImage` module to your workspace and select load *$(DemoDataPath)/BrainMultiModal/ProbandT1.dcm*. Add a `View2D` module and connect both as seen below.
+
+![Example Network](/images/tutorials/image_processing/network_example3.png "Example Network")
+
+### Add the RegionGrowing module
+Add the `RegionGrowing` module and connect the input with the `LocalImage` module. You will see a message *results invalid*. The reason is, that a region growing always needs a starting point for getting similar pixels. The output of the module does not show a result in *Output Inspector*.
+
+![Results Invalid](/images/tutorials/image_processing/network_example3a.png "Results Invalid")
+
+Add a `SoView2DMarkerEditor` to your network and connect it with your `RegionGrowing` and with the `View2D`. Clicking into your viewer now creates markers which can be used for the region growing. 
+
+![SoView2DMarkerEditor](/images/tutorials/image_processing/SoView2DMarkerEditor.png "SoView2DMarkerEditor")
+
+The region growing starts on manually clicking *Update* or automatically if you check *Update Mode* *Auto-Update*. We recommend to set update mode to automatically. Additionally you should set the *Neighborhood Relation* to *3D-6-Neighborhood (x,y,z)*, because then your segmentation will also affect the z-axis.
```

Comment:
if you check *Update Mode* *Auto-Update*. We recommend to set update mode to automatically ->
if *Update Mode* is set to *Auto-Update*. We recommend to set update mode to automatic update (or: to *Auto-Update*)